Bottom: 3bf16d1d38446c9c7b89d18bbc48902aa1bbbcab
Top:    51c87db29a00eecb08d6da970aaab359b6e220e5
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 14:06:34 +0800

Refresh of macvlan-xdp-rx-handler-support

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index cfda146..c54881b 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -435,6 +435,16 @@ static void macvlan_forward_source(struct sk_buff *skb,
 	}
 }
 
+int macvlan_handle_frame_xdp(struct xdp_buff *xdp)
+{
+	const struct ethhdr *hdr = (const struct ethhdr *)xdp->data;
+
+	if (is_multicast_ether_addr(eth->h_dest))
+		return -EFAULT;
+
+	
+}
+
 /* called under rcu_read_lock() from netif_receive_skb */
 static rx_handler_result_t macvlan_handle_frame(struct sk_buff **pskb)
 {
